lib/xdrgen/generators/rust.rb

```diff
@@ -134,6 +134,7 @@ def render_source_comment(out, defn)
 
       def render_struct(out, struct)
         out.puts "#[derive(Clone, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]"
+        out.puts "#[cfg_attr(feature = \"serde\", derive(Serialize, Deserialize), serde(rename_all = \"camelCase\"))]"
```

Comment:
It is common in Ruby to use alternative string style when the string will contain double quotes:
```suggestion
        out.puts %{#[cfg_attr(feature = "serde", derive(Serialize, Deserialize), serde(rename_all = "camelCase"))]}
```

Reply:
Done